### ISSUE TITLE
gq.com page truncated

### ISSUE THREAD

**Issue by sargon2** [postlight/mercury-parser#516]
When I load this page in Mercury Reader:

https://www.gq.com/story/mount-everest-chaos-at-the-top-of-the-world

The content is truncated at the first interstitial ad.

(Apologies for not using the issue template, but I deleted it out of the issue text box, and I don't see how to get it back -- I cleared all local content such as cookies but when I click new issue it's still gone.)

**Comment by markkerrigan** [comment]
I've noticed a similar issue too on another Conde Nast site. The article is truncated after the first ad.

https://www.vogue.com/article/body-hair-new-generation-not-shaving

**Comment by mcdickenson** [comment]
@sargon2 I've seen this same issue with a couple of pages, such as https://www.gq.com/story/the-great-buenos-aires-bank-heist and https://www.wired.com/story/the-secret-history-of-a-cold-war-mastermind/. Did you ever find a fix or work-around?

**Comment by sargon2** [comment]
@mcdickenson yeah, I have a workaround, but it's a lot of work -- download the whole page, edit the html locally to remove all non-content tags, load it off disk in the browser and run mercury there.

**Comment by mcdickenson** [comment]
@sargon2 That makes sense--thanks for sharing 👍
